example/config.make

```diff
@@ -88,7 +88,7 @@
 # add a runtime path to search for those shared libraries, since they aren't 
 # incorporated directly into the final executable application binary.
 # TODO: should this be a default setting?
-# PROJECT_LDFLAGS=-Wl,-rpath=./libs
+PROJECT_LDFLAGS=-lz -L /lib64 -l avahi-client -lavahi-common
```

Comment:
it's better it fhis flags went in an addon_config.mk file. if you want to take a look that would be great. if you don't know very well the structure i can take a look at some point.

thanks!!